src/components/Header.jsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
```

Comment:
- [ ] kindly Try to fix linters as much as possible linters are there to improve your application so I will highly recommend you remove this line and use it only to disable some unfixable without affecting the functionality of your application ⚒️⚒️😁